doc/api/deprecations.md

```diff
@@ -1058,14 +1073,17 @@ changes:
     description: Documentation-only deprecation.
 -->
 
-Type: Documentation-only
+Type: Runtime
 
 The [`util.isError()`][] API is deprecated.
```

Comment:
Shouldn't migration path be provided for all deprecated items? Currently some of them have it, some don't.

Reply:
they are just shortcuts for `typeof x === y`,  I assume is quite obvious

Reply:
It's not obvious, pointing that out would be helpful.

Reply:
also `typeof x === 'Error'` is not a thing, I think?.. is it instanceof?